bin/run-cloud

```diff
@@ -0,0 +1,15 @@
+ #! /usr/bin/env bash
+
+# DOC: Run deploy scripts from Docker.
+
+# Usage: 
+# bin/build-cloud
+# bin/run-cloud <command> <tag> <config_file>
+
+export COMMAND="${1:-deploy}"
+export IMAGE_TAG="${2:-latest}"
+export CONFIG="${3:-cloud/deploys/dev_aws/civiform_config.sh}"
+export AWS_ACCESS_KEY_ID=$(aws --profile default configure get aws_access_key_id)
+export AWS_SECRET_ACCESS_KEY=$(aws --profile default configure get aws_secret_access_key)
+echo "Running ${COMMAND} ${IMAGE_TAG} ${CONFIG}"
+docker compose -f cloud/docker-compose.yaml up
```

Comment:
There's some arguments you can add to remove and shut down the container once the commands are done.  Might be a good idea here

Reply:
couldn't find arguments, but I addded rm command right after this.